apps/site/src/app/apply/page.tsx

```diff
@@ -20,47 +20,53 @@ export default async function Apply({
 		prefaceAccepted?: string;
 	};
 }) {
+	const hasAcceptedQueryParam =
+		searchParams !== undefined && searchParams?.prefaceAccepted === "true";
```

Comment:
Partly out of scope, but since we're using optional chaining, is the first check for `undefined` actually necessary?

Reply:
You're right, the first check is not necessary.